monitor/src/main/java/edu/asu/diging/monitor/core/service/INotificationManager.java

```diff
@@ -9,5 +11,7 @@
 	boolean addRecipient(String name, String email) throws EmailAlreadyRegisteredException;
 
 	void sendNotificationEmails(IApp app, AppStatus previousStatus);
+	
+	Map<String,String> showRecipients();
```

Comment:
how about calling this `getRecipients` or `getAllRecipients` since this method doesn't show anything?